channels/gr.m3u

```diff
@@ -95,10 +95,6 @@ http://ert-live-bcbs15228.siliconweb.com/media/ert_world/ert_worldmedium.m3u8
 http://wpso.com:1936/hls/wzra.m3u8
 #EXTINF:-1 tvg-id="ERT1HD.gr" tvg-name="ERT1 HD" tvg-country="GR" tvg-language="" tvg-logo="https://i.imgur.com/Dd3HM0I.png" group-title="",ERT1 HD
 http://ert-live-bcbs15228.siliconweb.com/media/ert_1/ert_1medium.m3u8
```

Comment:
Replacement link (in case of) :  http://ert-live-bcbs15228.siliconweb.com/media/ert_1/ert_1.m3u8